error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/StringCaseLocaleUsageTest.java

```diff
@@ -54,7 +48,7 @@ void identification() {
 
   @Test
   void replacementFirstSuggestedFix() {
-    refactoringTestHelper
+    newInstance(StringCaseLocaleUsage.class, getClass())
```

Comment:
```suggestion
    BugCheckerRefactoringTestHelper.newInstance(StringCaseLocaleUsage.class, getClass())
```
Likewise below.

Reply:
Started working on that upstream fix. Will do that after applying this feedback.

Reply:
Opened: https://github.com/PicnicSupermarket/error-prone/pull/58.